### DISCUSSION TITLE
Complete Newbie Plotting Question.

### DISCUSSION THREAD

**Discussion by spacebetweenspace** [Chia-Network/chia-blockchain#1090] [Q&A]
Hello all,

     I've discovered the chia network recently via a friend and downloaded the chia blockchain for windows. I purchased a 12TB external hard drive and began to plot onto the hard drive (both the temporary space and final location). Forgive me for being a complete newb, I need help.
![Plots](https://user-images.githubusercontent.com/79780219/109409189-676f8c00-7945-11eb-91e5-ec9a69cb54d8.PNG)

Here are my specs

Acer Nitro 5 gaming laptop
Window 10 OS
Intel Core i5 8th Gen
8 GB of RAM
WD 12TB Elements Desktop Hard Drive 
  
A few questions. 
1. Do I need to download anything else besides the Chia Blockchain to start plotting?
2. How many 32k plots can I create at the same time?
3. Can you look at the picture I attached? Is it actually plotting right now or am I doing something wrong?
4. What exactly is plotting in parallel?

**Comment by dorkmo** [comment]
Under the "Action" column click the three dots and you can view the log.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
